README.md

```diff
@@ -0,0 +1,29 @@
+# SQLite Speedtest with Emscripten
+
+This repository shows how to compile and run the speedtest1 from the SQLite
+project in the browser using different storage backends.
+
+## Getting the Source
+
+```shell
+git clone --recurse-submodules https://github.com/fivedots/sqlite-speedtest.git
+```
+
+## Running the speedtest
+To compile, navigate to the sqlite-speedtest directory and run
+
+```shell
+make all
```

Comment:
This did not work for me, is there a step that I'm missing?

Error was:
"missing function: $stackTrace
-1
missing function: $stackTrace
-1
missing function: $stackTrace
-1
missing function: $stackTrace
-1
missing function: $stackTrace
-1
/tmp/tmpSkdVcB.js:127
      throw ex;
      ^

RangeError: Maximum call stack size exceeded
    at Object.Console.<computed> (internal/console/constructor.js:268:9)
    at Object.warn (internal/console/constructor.js:314:26)
    at abort (/tmp/tmpSkdVcB.js:1623:3)
    at stackTrace (/tmp/tmpSkdVcB.js:1851:41)
    at abort (/tmp/tmpSkdVcB.js:1628:44)
    at stackTrace (/tmp/tmpSkdVcB.js:1851:41)
    at abort (/tmp/tmpSkdVcB.js:1628:44)
    at stackTrace (/tmp/tmpSkdVcB.js:1851:41)
    at abort (/tmp/tmpSkdVcB.js:1628:44)
    at stackTrace (/tmp/tmpSkdVcB.js:1851:41)
FAIL: Running the generated program failed!
make: *** [Makefile:108: dist/sqlite3.js] Error 1
"

Reply:
This is odd, the new Makefile only has 47 lines, so there should not be an error in Line 108. Can you confirm that all files were pulled to this version?